doc/ref_arch/openstack/chapters/chapter08.md

```diff
@@ -1,7 +1,7 @@
 [<< Back](../../openstack)
 
 # 8. Gaps, Innovation, and Development
-<p align="right"><img src="../figures/bogo_dfp.png" alt="scope" title="Scope" width="35%"/></p>
+<p align="right"><img src="../figures/bogo_com.png" alt="scope" title="Scope" width="35%"/></p>
```

Comment:
@karinesevilla Will Chapter 8 be ever complete as it is about Gaps, Innovation, etc.? Maybe "Dickering over Fine Points"?
Depending upon what you decide, the "alt" and "title" will also need to be adjusted.

Reply:
@pgoyal01 Suggestion integrated

Reply:
Move to dickering over fine points